front/src/applications/operationalStudies/components/SimulationResults/SpaceTimeChart/SpaceTimeChart.jsx

```diff
@@ -226,6 +226,8 @@ export default function SpaceTimeChart(props) {
       );
       moveGridLinesOnMouseMove();
     }
+
+    dispatchUpdateChart(chart);
```

Comment:
Can you add a comment so no one will remove it by accident ?
```suggestion
    // required to sync the warpedMap
    dispatchUpdateChart(chart);
```